maps/005-maze.txt

```diff
@@ -1,9 +1,9 @@
 ###           #                      #H  |
 #P#           #                      ### |
 # #B          #                 #    #   |
-# ##  #  #BB# #              #  #    # ##|
+# ##  #  #BB  #              #  #    # ##|
 # ##  #  #  # # ##OO         #  #    #  b|
-B #B  #  #r # #    O         #  #    ####|
+B #B  #  #r # #    O         #  #    ##B#|
```

Comment:
I like this idea, but I'm not sure you could actually break this block as there's no space to jump above it. This could work but needs to be one space left.

Reply:
Good spot, yes. Perhaps I should have actually fired up the level rather than trusting my judgement here...